bad_code.java

```diff
@@ -8,14 +8,15 @@ public static void main(String[] args) {
 }
 
 class ParentClass {
-    public int someNumber = 10;
-    public String someString = "Hello";
+    public int number = 10;
```

Comment:
Use a more descriptive name for the 'number' variable, such as 'initialNumber'. [medium]